### PR TITLE
Add Netlify as sponsor

### PR DESCRIPTION
**Pull request by ericholscher**
----
📚 Documentation preview 📚: https://writethedocs-www--2105.org.readthedocs.build/

### CODE REVIEW

File: docs/_data/portland-2024-config.yaml [hunk 1]
```diff
@@ -184,7 +184,7 @@ sponsors:
       link: https://www.mongodb.com/?ref=writethedocs
       brand: MongoDB
     - name: jetbrains-svg
-      link: https://www.jetbrains.com/?ref=writethedocs
+      link: https://jebrains.com/writethedocs
```
Comment (plaindocs) [inline docs/_data/portland-2024-config.yaml:187]: Is this change intentional?

**Review by plaindocs — APPROVED** [review]

File: docs/_data/portland-2024-config.yaml (hunk 1 above)
Comment (ericholscher) [reply docs/_data/portland-2024-config.yaml:187]: Yep.
